torch/_torch_docs.py

```diff
@@ -2312,9 +2312,11 @@ def merge_dicts(*dicts):
     inv = (u^T u)^{{-1}}
 
 Args:
-    input (Tensor): the input 2-D tensor :math:`u`, a upper or lower triangular
-           Cholesky factor
-    upper (bool, optional): whether to return a lower (default) or upper triangular matrix
+    input (Tensor): the input tensor :math:A of size :math:`(*, n, n)`,
+                where * is zero or more batch dimensions consisting of
```

Comment:
It reads a bit as "batch dimensions consisting of symmetric PSD matrices", while it's supposed to be the "tensor consisting of symmetric PSD matrices", right? just a bit unclear

Reply:
Thanks for your feedback. I modified the description following the description of `cholesky_solve`.